### PR TITLE
Rename enabled to isarConnected

### PR DESCRIPTION
**Pull request by Eddasol**

### CODE REVIEW

File: frontend/src/api/ApiCaller.tsx [hunk 1]
```diff
@@ -134,7 +134,7 @@ export class BackendAPICaller {
     static async getEnabledRobots(): Promise<Robot[]> {
         const path: string = 'robots'
         const result = await BackendAPICaller.GET<Robot[]>(path).catch(BackendAPICaller.handleError('GET', path))
-        return result.content.filter((robot) => robot.enabled)
```
Comment (Eddasol) [inline frontend/src/api/ApiCaller.tsx:137]: Note that the name of this function (getEnabledRobots) hasn't been changed, as this function will be rewritten to sort on deprecated instead in separate PR

File: backend/api/Services/RobotService.cs [hunk 2]
```diff
@@ -268,7 +268,7 @@ public async Task<IList<Robot>> ReadRobotsForInstallation(string installationCod
                 .ToListAsync();
         }
 
-        public async Task SetRobotOffline(string robotId)
+        public async Task SetRobotToIsarDisconnected(string robotId)
```
Comment (oysand) [inline backend/api/Services/RobotService.cs:271]: This function does more than it says, we split the functionality or set the function to explicitly state what it does

Comment (Eddasol) [reply backend/api/Services/RobotService.cs:271]: Handled by issue https://github.com/equinor/flotilla/issues/1493

**Review by oysand — APPROVED** [review]
LGTM

**Pull request merged**